scss/mixins/_grid.scss

```diff
@@ -3,10 +3,13 @@
 // Generate semantic grid columns with these mixins.
 
 @mixin make-row($gutter: $grid-gutter-width) {
+  --grid-gutter-x: #{$gutter};
+  --grid-gutter-y: 0;
```

Comment:
As a sidenote, you could remove this declaration and use `var(--grid-gutter-y, 0)` in `margin-top` to shorten things a bit. Could be useful in some cases to reduce custom properties bloat.

Not sure if it's readable enough, though.

Reply:
We do need to reset this custom property, otherwise styles will leak in to nested rows.

Reply:
OW nested things. I need to accomodate :D